axelar-chains-config/info/mainnet.json

```diff
@@ -2007,6 +2007,43 @@
         "url": "https://explorer.immutable.com",
         "api": "https://explorer.immutable.com/api"
       }
+    },
+    "fraxtal": {
+      "name": "Fraxtal",
+      "id": "fraxtal",
+      "axelarId": "fraxtal",
+      "chainId": 252,
+      "rpc": "https://rpc.frax.com",
+      "tokenSymbol": "frxETH",
+      "confirmations": 2,
+      "gasOptions": {
+        "gasLimit": 8000000
```

Comment:
this was set to 3mm originally, but i pushed to 8mm as set on other chains. not sure if 3mm was intentional

Reply:
3m was able to run all tests successfully for me and 8m resulted in the lack of funds for execution, therefore committed 8m. Good with 8m here for consistency.